### ISSUE TITLE
Consider adding MIT license 

### ISSUE THREAD

**Issue by mattysweeps** [precious/bash_minifier#7]
Please consider adding the MIT license to the project. Thank you

**Comment by thetayloredman** [comment]
> Please consider adding the MIT license to the project. Thank you

Yeah. Currently it is **copyright fraud** to use this project as it is licenceless aka under exclusive implied copyright.
